src/App.js

```diff
@@ -1,24 +1,46 @@
 import React from 'react';
-import './App.css';
+import { Tab } from './components/Tab/Tab';
+import 'bulma/css/bulma.css';
 
-// eslint-disable-next-line no-unused-vars
 const tabs = [
   {
-    title: 'Tab 1',
-    content: 'Some text 1',
+    title: 'Tab 1', content: 'Some text 1',
   },
   {
-    title: 'Tab 2',
-    content: 'Some text 2',
+    title: 'Tab 2', content: 'Some text 2',
   },
   {
-    title: 'Tab 3',
-    content: 'Some text 3',
+    title: 'Tab 3', content: 'Some text 3',
   },
 ];
 
-const App = () => (
-  <h1>React tabs</h1>
-);
+class App extends React.Component {
+  state = {
+    active: this.selectTab,
+  }
+
+  selectTab = (e) => {
+    this.setState({
+      active: e,
+    });
+  };
+
+  render() {
+    return (
+      <div className="App">
+        <h1>React tabs</h1>
+        <Tab
+          tabs={tabs}
+          active={this.state.active}
+          handlerOnClick={this.selectTab}
+        />
+        <div className="content">
+          {tabs.filter(tab => tab.title === this.state.active)
+            .map(tab => <p>{tab.content}</p>)}
```

Comment:
if you save selected tab's index in state you won't need to filter tabs in order to find one
you can just `tabs[this.state.selectedTabIndex].content`